mock2/private/scss/footer/footer.scss

```diff
@@ -0,0 +1,33 @@
+.footer {
+  height: 100px;
+  width: 100%;
+  margin: 0 6%;
```

Comment:
It is not good idea to use margins for BEM's block, especially margin with percentage

Reply:
my bad, missed it.